packages/x-license/src/utils/licenseScope.ts

```diff
@@ -1,3 +1,20 @@
 export const LICENSE_SCOPES = ['pro', 'premium'] as const;
+export const PRODUCT_SCOPES = ['data-grid', 'date-pickers', 'charts', 'tree-view', null] as const;
+export const PLAN_VERSIONS = ['legacy', 'Q3-2024'] as const;
```

Comment:
Nitpick

I would advocate against using `legacy` because it is relative to when we are in time.
What is not legacy today might become legacy tomorrow, so we will end up with licenses that are labelled `Q3-2024` or even `Q2-2026` that are legacy (meaning that they are only for grandfathered customers at the time of the discussion) but not marked as "legacy" since "legacy" in the license means "legacy as of when we introduced the property).

I would prefer something like "initial" that still have this idea of being older but without the relative aspect (the initial version of the plan will still be the one we have before Q3 2024 even if we have 4 other iterations on the pricing), or maybe the date of the first launch of the pro plan (Q3-2020 I think).

Reply:
makes sense